web/src/main/java/org/akaza/openclinica/control/admin/DeleteEventCRFServlet.java

```diff
@@ -240,21 +236,21 @@ private void createDiscrepancyNoteBean(String description, String detailedNotes,
         dnb.setDescription(description);
         dnb.setDetailedNotes(detailedNotes);
         dnb.setDiscrepancyNoteTypeId(parentDiscrepancyNote.getDiscrepancyNoteTypeId()); // set to parent DN Type Id
-        dnb.setResolutionStatusId(4); // set to closed
+        dnb.setResolutionStatusId(6); // set to closed-modified
         dnb.setColumn("value"); // this is needed for DN Map object
-        dnb.setAssignedUserId(ub.getId());
+        dnb.setAssignedUser(null);
         dnb.setOwner(ub);
         dnb.setParentDnId(parentDiscrepancyNote.getId());
         dnb.setActivated(false);
         dnb = (DiscrepancyNoteBean) getDnDao().create(dnb); // create child DN
         getDnDao().createMapping(dnb); // create DN mapping
 
         DiscrepancyNoteBean itemParentNote = (DiscrepancyNoteBean) getDnDao().findByPK(dnb.getParentDnId());
-        itemParentNote.setResolutionStatusId(ResolutionStatus.CLOSED.getId());
-        itemParentNote.setAssignedUserId(ub.getId());
+        itemParentNote.setResolutionStatusId(6); // set to closed-modified
```

Comment:
I know it is existing code but is there any way to remove these hard coded numbers for statuses?